src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1015,8 +1028,19 @@
           "type": "integer"
         },
 
-        "configurationOverrides": {
-          "description": "An array of configurationOverride objects that describe runtime reporting behavior.",
+        "ruleConfigurationOverrides": {
+          "description": "An array of configurationOverride objects that describe rules related runtime overrides.",
+          "type": "array",
+          "minItems": 0,
+          "default": [],
```

Comment:
this is wrong, minItems s/be 1 in both this case and notificationConfigurationOverrides.

Ask yourself: what value does an empty collection provide? this is very rare. mostly, it shouldn't be present at all, or there should be at least one item in the array.